app/assets/stylesheets/kudos/components/_give-kudo.scss

```diff
@@ -3,8 +3,105 @@
   @extend .container;
   margin-bottom: $spacer;
   width: $give-kudo-width;
+  font-family: $font-gotham;
+  font-weight: 500;
+  text-align: center;
+}
+
+.give-kudo__modal {
+  text-align: left;
+  position: fixed;
+  z-index: 99 !important;
+  background: white;
+  border-radius: 4px;
+  padding: 35px 30px 35px 30px;
```

Comment:
There's a handy [`padding` shorthand](https://developer.mozilla.org/en-US/docs/Web/CSS/padding) you can use for vertical/horizontal padding!
```css
foo {
  padding: 35px 30px;
}
```